pages/common/shar.md

```diff
@@ -0,0 +1,28 @@
+# shar
+
+> Create a shell archive.
+> More information: <https://manned.org/shar>.
+
+- Specify compression for input files:
+
+`shar --intermix-type`
+
+- Specify compaction (compression) program:
+
+`shar --compactor={{program}}`
+
+- Pass level for compression:
+
+`shar --level-of-compression={{level}}`
```

Comment:
```suggestion
`shar --level-of-compression={{1,...,9}}`
```

Optional fix: `{{level}}` can be replaced with valid level number range.

Reply:
> ```suggestion
> `shar --level-of-compression={{1,...,9}}`
> ```
> 
> Optional fix: `{{level}}` can be replaced with valid level number range.

How about replacing it with 1,2,3,4,5,6,7,8,9 and if there are a lot of numbers we can use a number range.

Reply:
> How about replacing it with 1,2,3,4,5,6,7,8,9 and if there are a lot of numbers we can use a number range.

Such format can take place. But IMO it's better to use range based format when we a referring to more than 2 number range.